src/platform/site-wide/cta-widget/helpers.js

```diff
@@ -1,7 +1,7 @@
-import backendServices from '../../user/profile/constants/backendServices';
-import environment from '../../utilities/environment';
+import backendServices from 'platform/user/profile/constants/backendServices';
```

Comment:
the cleaner imports make me unreasonably happy 😁

Reply:
well, Happy Friday too then!  :D